modules/pytket-qulacs/tests/test-requirements.txt

```diff
@@ -1 +1,3 @@
-git+git://github.com/quantumlib/OpenFermion@086073666d275de736861b41ed535a9fe75e63ac#egg=openfermion
+hypothesis==6.14.3
+pytest==6.2.4
+openfermion==1.0.0
```

Comment:
We should use openfermion 1.0.1. The reason for selecting the commit above was that 1.0.0 was not compatible with cirq 0.11, but 1.0.1 should fix that. Also can we say `~=` rather than `==`?

Reply:
I have made this change. 

Specifying the dependency as:
"OpenFermion~=1.0.0"

installs v1.0.1 as suggested